javatests/arcs/android/entity/DifferentAndroidHandleManagerDifferentStoresTest.kt

```diff
@@ -59,9 +66,4 @@ class DifferentAndroidHandleManagerDifferentStoresTest : HandleManagerTestBase()
     // TODO - fix these?
     override fun collection_referenceLiveness() {}
     override fun singleton_referenceLiveness() {}
-
-    // We don't expect these to pass, since Operations won't make it through the driver level
-    override fun singleton_writeAndOnUpdate() {}
-    override fun collection_writeAndOnUpdate() {}
```

Comment:
In the Android configurations, these *will* pass, since the storage service uses the same stores, even though the ServiceStore instances are different.